pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -6348,40 +6348,45 @@ func TestChangefeedOnlyInitialScan(t *testing.T) {
 
 		for testName, changefeedStmt := range initialScanOnlyTests {
 			t.Run(testName, func(t *testing.T) {
-				sqlDB.Exec(t, "CREATE TABLE foo (a INT PRIMARY KEY)")
-				sqlDB.Exec(t, "INSERT INTO foo VALUES (1), (2), (3)")
+				sqlDB.Exec(t, `CREATE TABLE foo (a INT PRIMARY KEY)`)
+				sqlDB.Exec(t, `INSERT INTO foo (a) SELECT * FROM generate_series(1, 5000);`)
```

Comment:
All it took to reproduce this issue was to go from 3 values to 1000 in the original test, yet this didn't flake on 3 (as far as I know).  A good reminder that there's distinct value in doing larger tables in our unit tests.